docs/index.md

```diff
@@ -1422,7 +1422,7 @@ The "JSON" reporter outputs a single large JSON object when the tests have compl
 
 ### JSON Stream
 
-The "JSON stream" reporter outputs newline-delimited JSON "events" as they occur, beginning with a "start" event, followed by test passes or failures, and then the final "end" event.
+The "JSONStream" reporter outputs newline-delimited JSON "events" as they occur, beginning with a "start" event, followed by test passes or failures, and then the final "end" event.
```

Comment:
How about to add `json-stream` as well? Like: 

```
"JSONStream"(or `json-stream`)
```

Reply:
Sure!  Including the alternates would be good too.  Since the way we actually use "JSONStream" and "json-stream" is identical, should they both be quoted/code-blocked the same way?

eg. The "JSONStream" (or "json-stream") reporter outputs...
eg. The `JSONStream` (or `json-stream`) reporter outputs...

Reply:
I like a code-blocked style.

Reply:
I have created a separate issue for the formatting as it is beyond the scope of the this PR.
https://github.com/mochajs/mocha/issues/4054